app/assets/javascripts/govuk_publishing_components/accessibility-test.js

```diff
@@ -7,28 +7,35 @@
     }
 
     if (!document.querySelector(selector)) {
-      return callback('No selector "' + selector + '" found')
+      return callback()
     }
 
     var axeOptions = {
       restoreScroll: true,
       include: [selector]
     }
 
+    // TODO: Remove when aXe core patched
+    // https://github.com/dequelabs/axe-core/issues/525
+    if (document.querySelector('svg') && !(document.querySelector('svg').children instanceof HTMLCollection)) {
+      delete axeOptions['restoreScroll']
+    }
+
     axe.run(selector, axeOptions, function (err, results) {
       if (err) {
-        callback('aXe Error: ' + err)
+        return callback('aXe Error: ' + err)
       }
 
-      var violations = (typeof results === 'undefined') ? [] : results.violations
-      var incompleteWarnings = (typeof results === 'undefined') ? [] : results.incomplete
-
-      if (violations.length === 0 && incompleteWarnings.length === 0) {
-        return callback('No accessibility issues found')
+      if (typeof results === "undefined") {
+        return callback('aXe Error: Expected results but none returned')
       }
 
-      var incompleteWarningsObj = _processIncompleteWarnings(incompleteWarnings)
-      var errorText = _processViolations(violations, results.url)
+      var errorText = _processViolations(results.violations, results.url)
+      var incompleteWarningsObj = _processIncompleteWarnings(results.incomplete)
+
+      var bodyClass = results.violations.length === 0 ? "test-a11y-success" : "test-a11y-failed"
+      document.body.classList.add(bodyClass);
+      document.body.classList.add("test-a11y-finished");
```

Comment:
Rather than this can we assign a global maybe?

```window.__A11Y_TEST_STATUS = 'finished'```

Reply:
Ah you want to use them in integration tests... If it's not easy to do the above seems fine. Might want to add 'js-' though.

Reply:
I think globals will work too. :+1: